translator/util/sdkutil.go

```diff
@@ -36,14 +35,15 @@ func DetectAgentMode(configuredMode string) string {
 		fmt.Println("I! Detected from ENV instance is EC2")
 		return config.ModeEC2
 	}
-
+	
 	if defaultEC2Region() != "" {
 		fmt.Println("I! Detected the instance is EC2")
 		return config.ModeEC2
 	}
 
 	if defaultECSRegion() != "" {
 		fmt.Println("I! Detected the instance is ECS")
+		fmt.Printf("I! Cluster: %s, region: %s \n",defaultECSRegion(),defaultClusterName())
```

Comment:
The values don't map to the right things. The format string is Cluster, Region, but the parameters passed in are in the order of Region, Cluster

Reply:
Sorry forgot to delete it. Mostly it for my testing on other PR.